### ISSUE TITLE
How to change the truncation rules?

### ISSUE THREAD

**Issue by boxed** [getsentry/sentry-docs#1053]
This document spells out the truncation rules: https://docs.sentry.io/development/sdk-dev/data-handling/ but I'm interested in changing those limits, and I can't find any reference to how to do this in the documentation. This is for an on-site installation.

**Comment by boxed** [comment]
This is very much still a problem. Maybe someone at TriOptima wants to keep an eye on it though. @tltx maybe

**Comment by boxed** [comment]
Yes. Still a problem.

**Comment by HazAT** [comment]
@boxed These limits are spread across our codebase and are not really meant to be changed. Sometimes they are even defined in the database. If you want to change specific ones the only thing I can tell you is, you need to do the work yourself, find the value you want to change and increase the limit.

**Comment by boxed** [comment]
Do the limits at least follow some kind of naming convention in the code base like `BLABLA_LENGTH_LIMIT`? Or is it just many hard coded magic numbers?

**Comment by HazAT** [comment]
I am afraid magic numbers. The easiest really would be: What property are you trying to increase? Find it within our models https://github.com/getsentry/sentry/tree/master/src/sentry/models and work your way back.

Sorry :/

**Comment by boxed** [comment]
Well actually I have since changed jobs :P But this really was a pervasive problem in production. Very often sentry would throw away the piece of information we desperately needed to see what the problem was. If this was an intermittent bug that we couldn't reproduce and happened with weeks apart this was extremely frustrating. 

We had lots of problems like this with sentry actually. Some stack frames didn't have local variables for example. And there's the suggestion to expand accessed variables on `self` that were accessed at the crash point that I wrote a POC for that was just closed.

The point of sentry to me is to show me information so I can fix problems...
